### DISCUSSION TITLE
Connection Limit in React Flow using Custom Nodes

### DISCUSSION THREAD

**Discussion by sudarsanarao995** [xyflow/xyflow#3525] [General]
For Connection Limit we are following below link 

https://reactflow.dev/docs/examples/nodes/connection-limit/

here codesandbox link 
https://codesandbox.io/s/vpgkmh?file=/CustomNode.js&utm_medium=sandpack

Here  we want Node 1 and Node 2 both nodes should be a **Custom Nodes** and make sure Connection Limit should be happened

![image](https://github.com/wbkd/react-flow/assets/147405464/95d38a8a-4f01-4065-8c62-d07df57f4a0f)

**Comment by bcakmakoglu** [comment]
What's the question though? 🤔

**Reply by sudarsanarao995** [reply]
Yes, basically as in document  Node 1 and Node 2 are in input type nodes , so is it possible to make Node 1 and Node 2 as a Custom Nodes to implement as Connection Limit

**Reply by bcakmakoglu** [reply]
Yeah, sure - everything that is built in the example can be used in a custom node as well :)
